<?xml version="1.0" ?>
<sdf version="1.6">
  <model name="tail_marker">
    <pose>0 0 0 0 0 0</pose>
    <static>true</static>
    <link name="link">
      <visual name="visual">
        <geometry>
          <sphere>
            <radius>0.05</radius>
          </sphere>
        </geometry>
        <material>
          <ambient>1 1 0 1</ambient>
          <diffuse>1 1 0 1</diffuse>
        </material>
      </visual>
    </link>
  </model>
</sdf>
